libkbfs/folder_branch_ops.go

```diff
@@ -5950,6 +5950,7 @@ func (fbo *folderBranchOps) unstageAfterFailedResolution(ctx context.Context,
 	default:
 	}
 
+	ctx = newLinkedContext(ctx)
```

Comment:
Can you please add a comment here explaining why this is needed, with a mention of KBFS-2150?  Why is this the right fix, rather than archiving any unmerged pointers as I suggested in the ticket?